drivers/input/touchscreen/synaptics_dsx_i2c.c

```diff
@@ -3809,11 +3857,14 @@ static void synaptics_rmi4_set_params(struct synaptics_rmi4_data *rmi4_data)
 	set_bit(KEY_HOMEPAGE, rmi4_data->input_dev->keybit);
 	set_bit(KEY_F3, rmi4_data->input_dev->keybit);
 	set_bit(KEY_DOUBLE_TAP, rmi4_data->input_dev->keybit);
```

Comment:
I thought that mce could also handle a power key event generated by the touch screen. That and the fact that it was not disabled in the onyx kernel let me decide to leave it enabled.